app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+  <div class="container-fluid">
+    <% if root_path %>
+      <%= link_to path do %>
+        <i class="fa fa-arrow-left  text-white"></i>
+      <% end %>
+    <% else %>
+      <i class="fa fa-bars text-white">
+      </i>
+    <% end %>
+    <a class="navbar-brand" href="#"><%= text %></a>
+    <% if current_user.present? %>
+      <%= button_to destroy_user_session_path, class:"btn", method: :delete do %>
+        <i class="fa fa-power-off text-danger"></i>
+      <% end %>
+    <% else %>
+      <i class="fa fa-user text-white"></i>
+    <% end %>
+  </div>
```

Comment:
- [ ] Please know that the back button for some reason took me back to the splash page as if I logged out, I recommend that you hide it entirely when you're on the home page so you're safe from these issues.

  ![image](https://user-images.githubusercontent.com/20567503/209167655-0f2af6b7-c0df-471a-806a-34ea56ac3534.png)